sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -189,13 +190,15 @@ private[hive] class IsolatedClientLoader(
     (name.startsWith("com.google") && !name.startsWith("com.google.cloud")) ||
     name.startsWith("java.") ||
     name.startsWith("javax.sql.") ||
+    name.startsWith("org.apache.derby.") ||
```

Comment:
I think this shouldn't be added without a strong reason as discussed at https://github.com/apache/spark/pull/20944#issuecomment-481962005 if I am not mistaken.

Reply:
Can we try to find the cause and backport it within Hive side because it looks a bug in Hive 2.3.4? We can put some inputs into Hive side to make 2.3.5. It's mush smaller job for Hive side instead of dealing with HIVE-16391.

Reply:
If there's some delays during ^, we can discuss to consider this as a temporary bandaid fix but I am worried if we happen to introduce some issues hard to debug (like I met in Livy).

Reply:
We could fix it at Hive side: https://issues.apache.org/jira/browse/HIVE-21639